settings.gradle

```diff
@@ -211,6 +211,9 @@ project(':extensions-classgraph').projectDir = file('extensions/classgraph')
 include(':extensions-suanshu')
 project(':extensions-suanshu').projectDir = file('extensions/suanshu')
 
+include(':plugin')
+include(':plugin:figure')
```

Comment:
The difference here is that `:plugin` is a real project (whereas in a lot of the other cases, it was an empty project); happy to change to `:plugin-<x>` though

Reply:
web and proto used to sort of have that convention, kind of, but neither follows it any more (in opposite directions)

In contrast, server and grpc-api before it never did - we had both :grpc-api and :grpc-api-server-native/docker etc.

As far as I can tell, gradle considers a directory without an explicit project in it to be exactly the same as a directory with a project in it (which is why :proto and its children still work even though proto/build.gradle doesnt exist) - in fact you can omit the build.gradle and still have a project in a directory (it will be considered by project.subprojects, etc).